src/currency/iso_currencies.rs

```diff
@@ -1826,6 +1826,190 @@ pub mod iso {
+                "MNT" => Ok(MNT),
+                "MOP" => Ok(MOP),
+                "MRU" => Ok(MRU),
+                "MUR" => Ok(MUR),
+                "MVR" => Ok(MVR),
+                "MWK" => Ok(MWK),
+                "MXN" => Ok(MXN),
+                "MYR" => Ok(MYR),
+                "MZN" => Ok(MZN),
+                "NAD" => Ok(NAD),
+                "NGN" => Ok(NGN),
+                "NIO" => Ok(NIO),
+                "NOK" => Ok(NOK),
+                "NPR" => Ok(NPR),
+                "NZD" => Ok(NZD),
+                "OMR" => Ok(OMR),
+                "PAB" => Ok(PAB),
+                "PEN" => Ok(PEN),
+                "PGK" => Ok(PGK),
+                "PHP" => Ok(PHP),
+                "PKR" => Ok(PKR),
+                "PLN" => Ok(PLN),
+                "PYG" => Ok(PYG),
+                "QAR" => Ok(QAR),
+                "RON" => Ok(RON),
+                "RSD" => Ok(RSD),
+                "RUB" => Ok(RUB),
+                "RWF" => Ok(RWF),
+                "SAR" => Ok(SAR),
+                "SBD" => Ok(SBD),
+                "SCR" => Ok(SCR),
+                "SDG" => Ok(SDG),
+                "SEK" => Ok(SEK),
+                "SGD" => Ok(SGD),
+                "SHP" => Ok(SHP),
+                "SKK" => Ok(SKK),
+                "SLL" => Ok(SLL),
+                "SOS" => Ok(SOS),
+                "SRD" => Ok(SRD),
+                "SSP" => Ok(SSP),
+                "STD" => Ok(STD),
+                "STN" => Ok(STN),
+                "SVC" => Ok(SVC),
+                "SYP" => Ok(SYP),
+                "SZL" => Ok(SZL),
+                "THB" => Ok(THB),
+                "TJS" => Ok(TJS),
+                "TMT" => Ok(TMT),
+                "TND" => Ok(TND),
+                "TOP" => Ok(TOP),
+                "TRY" => Ok(TRY),
+                "TTD" => Ok(TTD),
+                "TWD" => Ok(TWD),
+                "TZS" => Ok(TZS),
+                "UAH" => Ok(UAH),
+                "UGX" => Ok(UGX),
+                "USD" => Ok(USD),
+                "UYU" => Ok(UYU),
+                "UYW" => Ok(UYW),
+                "UZS" => Ok(UZS),
+                "VES" => Ok(VES),
+                "VND" => Ok(VND),
+                "VUV" => Ok(VUV),
+                "WST" => Ok(WST),
+                "XAF" => Ok(XAF),
+                "XAG" => Ok(XAG),
+                "XAU" => Ok(XAU),
+                "XBA" => Ok(XBA),
+                "XBB" => Ok(XBB),
+                "XBC" => Ok(XBC),
+                "XBD" => Ok(XBD),
+                "XCD" => Ok(XCD),
+                "XDR" => Ok(XDR),
+                "XOF" => Ok(XOF),
+                "XPD" => Ok(XPD),
+                "XPF" => Ok(XPF),
+                "XPT" => Ok(XPT),
+                "XTS" => Ok(XTS),
+                "YER" => Ok(YER),
+                "ZAR" => Ok(ZAR),
+                "ZMK" => Ok(ZMK),
+                "ZMW" => Ok(ZMW),
+                "ZWL" => Ok(ZWL),
+
+                _ => Err(MoneyError::InvalidCurrency)
+            }
```

Comment:
```suggestion
            find_by_num_code(s).ok_or(MoneyError::InvalidCurrency)
```

Directly reuse `find_by_num_code` that simply the maintenance.